File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.js
/**
 * @param {number} n
 * @return {number}
 */
var mirrorDistance = function(n) {
    let rem=0;
    let ori=n;
   while(n!=0)
   {
     rem=rem*10+n%10;
    n=Math.floor(n/10);
   } 
   return Math.abs(ori-rem);
};

File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.py
class Solution(object):
    def mirrorDistance(self, n):
        """
        :type n: int
        :rtype: int
        """
        rem=0
        ori=n
        while not n==0:
        
          rem=rem*10+n%10
          n//=10
        
        return abs(ori-rem)